### DISCUSSION TITLE
How to write the project name?

### DISCUSSION THREAD

**Discussion by buhtz** [rfjakob/gocryptfs#872] [Q&A] [answered]
Looking into your docu and readme file it seems that the official name of the project (or name of the application) is written in lowercase letters. Is this correct?

So writing "GoCryptFS" wouldn't be correct in the meaning of "official".

Do you have an official statement about it? 😄

**Comment by kaoneko** [comment]
Pending an official statement, I think your observation is correct. Even at the beginning of a sentence gocryptfs is consistently written in all-lowercase, this must be a conscious choice.

Are you writing a Wikipedia page on gocryptfs?

**Reply by buhtz** [reply]
Thanks for the reply.
I am member of maintenance team of [Back In Time](https://github.com/bit-team/backintime). We do migrate ([PR #1897](https://github.com/bit-team/backintime/pull/1897)) from EncFS to gocryptfs and just wondered how to spell that name.

**Comment by rfjakob** [comment, marked as answer]
Yes the preferred name is "gocryptfs" (all lowercase)
